LoRDeckCodes_py3/lor_deck_encoder.py

```diff
@@ -0,0 +1,243 @@
+from base64 import b32encode, b32decode
+
+from card_code_and_count import CardCodeAndCount
+import varint_translator
+
+CARD_CODE_LENGTH = 7
+faction_to_int = {"DE": 0, "FR": 1, "IO": 2, "NX": 3, "PZ": 4, "SI": 5}
+int_to_faction = {0: "DE", 1: "FR", 2: "IO", 3: "NX", 4: "PZ", 5: "SI"}
+MAX_KNOWN_VERSION = 1
+
+
+def get_deck_from_code(code):
+    result = []
+
+    try:
+        bytes_ = b32decode(code + ('=' * (56 - len(code))))
```

Comment:
i'm curious, what's the significance of the trailing underscore in `bytes_` here?

Reply:
Avoids conflict with the [built-in python class](https://docs.python.org/3/library/stdtypes.html#bytes)

Reply:
ahh, in that case it might be preferable to just a more specific name instead of adding an underscore to avoid shadowing. something like `deck_code_bytes` perhaps?

Reply:
hmm, well, it's not the only variable with the trailing underscore dodge. so could leave it as it is, or change the others too, for consistency?

Reply:
considering you named it for consistency with the original implementation, i feel like the existing naming is fine :) if you're happy with it, than i am too.